.github/labels.yml

```diff
@@ -0,0 +1,71 @@
+# libevm specific labels
+- name: "Status: 🔴 Blocked by Geth sync"
+  color: "959a9c"
+  description: "This needs to wait for the next fork sync with the geth repository 🍴"
+- name: "Status: 🔴 DO NOT MERGE"
+  color: "959a9c"
+  from_name: "DO NOT MERGE"
+
+# General categories
+- name: "Category: New feature 🆕"
+  color: "ffc7ea"
+- name: "Category: Documentation ✒️"
+  description: "A problem with the readme or a code comment."
+  color: "ffc7ea"
+- name: "Category: Bug 🐛"
+  color: "ffc7ea"
+  from_name: "bug"
+- name: "Category: Investigation 🔍"
+  color: "ffc7ea"
+- name: "Category: CI ⛓️"
+  description: "Anything related to continuous integration"
+  color: "ffc7ea"
+- name: "Category: Testing 🧪"
+  description: "Unit tests, integration tests, etc."
+  color: "ffc7ea"
+- name: "Category: Code quality ✏️"
+  description: "Unit tests, integration tests, etc."
+  color: "ffc7ea"
+- name: "Category: Performance 🚀"
+  color: "ffc7ea"
+- name: "Category: Label missing ❗"
+  description: "Use this if it feels a label should be added to label this issue"
+  color: "ffc7ea"
+
+# Status
+- name: "Status: 🗯️ Waiting for feedback"
+  color: "f7d692"
+- name: "Status: 🔴 Blocked"
+  color: "f7d692"
+  description: "Blocked by another issue or pull request"
+- name: "Status: 🟡 Nearly resolved"
+  color: "f7d692"
+  description: "This might be resolved or is about to be resolved"
+
+# Priority
+- name: "Priority: 🚨 Urgent"
+  color: "03adfc"
+- name: "Priority: 📌 Before next release"
+  color: "03adfc"
+  description: "Has to be done before the next release"
+- name: "Status: 🔒 After next release"
+  color: "03adfc"
+  description: "Will be done after the next release"
+- name: "Priority: 💤 Low priority"
+  color: "03adfc"
+
+# Complexity
+- name: "Complexity: 🟥 Hard to do"
+  color: "ff9efc"
+- name: "Complexity: 🟩 Easy to do"
+  color: "ff9efc"
+
+# Closing reason
+- name: "Closed: 👥 Duplicate"
+  color: "959a9c"
+  description: "Issue duplicates an existing issue"
+- name: "Closed: ⚰️ Inactive"
+  color: "959a9c"
+  description: "No answer was received for weeks"
+- name: "Closed: 🗑️ Bad issue"
```

Comment:
When would this be used? Feels a bit passive aggressive 😬

Reply:
😄 It does, it would be for example if the issue is reported but it's irrelevant to the project (i.e. internet does not work for the user). Now as a library, we could probably remove that one since we won't have much of the public creating "bad issues", it'll mostly be internal first. REMOVED!